### DISCUSSION TITLE
When script is launch, all is OK, what happened ?

### DISCUSSION THREAD

**Discussion by sypqys** [undergroundwires/privacy.sexy#332] [Q&A] [answered]
Hello !

press a key to continue, and the command prompt closed...
I have to restart ?

![G34chBbEzW](https://github.com/undergroundwires/privacy.sexy/assets/27967268/6d420657-cbb5-4805-9739-1b730f68b847)

Thanks !!

**Comment by sypqys** [comment]
[script.txt](https://github.com/undergroundwires/privacy.sexy/files/14630704/script.txt)

**Comment by plantindesk** [comment, marked as answer]
**System Restart:** Use `shutdown /r` for restarting system.
